nuget/lib/dependabot/nuget/update_checker/repository_finder.rb

```diff
@@ -166,6 +167,20 @@ def repos_from_config_file(config_file)
               }
             end
 
+          disabled_sources =
+            doc.css("configuration > disabledPackageSources > add").map do |node|
```

Comment:
I think we could extract a method for this, the `repos_from_config` method is already pretty complext:

```ruby
        def disabled_sources # rubocop:disable Metrics/PerceivedComplexity
          @disabled_sources ||=
            doc.css("configuration > disabledPackageSources > add").map do |node|
              value = node.attribute("value")&.value ||
                      node.at_xpath("./value")&.content

              if value&.strip&.downcase == "true"
                node.attribute("key")&.value&.strip ||
                  node.at_xpath("./key")&.content&.strip
              end
            end
        end
```

I also made a small optimization by calling `&.strip&.downcase` on `value`

Reply:
Yes, I agree, that is much better, updated the PR with your suggested changes.